src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerAddAudioRequestPlaybackIntTest.java

```diff
@@ -73,11 +72,11 @@ void beforeEach() {
             SOME_COURTROOM,
             LocalDateTime.parse(HEARING_DATETIME)
         );
-        CourtCaseEntity courtCase = hearingEntity.getCourtCase();
-        courtCase.addProsecutor("aProsecutor");
-        courtCase.addDefendant("aDefendant");
-        courtCase.addDefence("aDefence");
-        dartsDatabase.save(courtCase);
+        //        CourtCaseEntity courtCase = hearingEntity.getCourtCase();
+        //        courtCase.addProsecutor("aProsecutor");
+        //        courtCase.addDefendant("aDefendant");
+        //        courtCase.addDefence("aDefence");
+        //        dartsDatabase.save(courtCase);
```

Comment:
Just checking if this commenting is intentional.

Reply:
same